### PR TITLE
Add likes to the PictureAdapter

### PR DESCRIPTION
**Pull request by aelalamy42**
photo1684510220
This PR adds a like counter to every post.
It also fixes a little notification problem

### CODE REVIEW

File: app/src/main/java/ch/epfl/culturequest/social/PictureAdapter.java [hunk 1]
```diff
@@ -125,11 +125,32 @@ public void onBindViewHolder(@NonNull PictureViewHolder holder, int position) {
...
         handleLike(holder, post);
         handleDelete(holder, post);
     }
 
+    /**
+     * Adapts the like count string to the number of likes.
+     *
+     * @param likes the number of likes
+     */
+    private String getNumberOfLikes(int likes) {
```
Comment (TheGuyWithoutH) [inline app/src/main/java/ch/epfl/culturequest/social/PictureAdapter.java:141]: Good job! It is going to help us in terms of responsive design

File: app/src/main/java/ch/epfl/culturequest/social/PictureAdapter.java [hunk 2]
```diff
@@ -125,11 +125,32 @@ public void onBindViewHolder(@NonNull PictureViewHolder holder, int position) {
 
         holder.location.setText("Lausanne");
 
+        // Set the like count
+        holder.likeCount.setText(getNumberOfLikes(post.getLikes()));
```
Comment (TheGuyWithoutH) [inline app/src/main/java/ch/epfl/culturequest/social/PictureAdapter.java:129]: I am just thinking of what the like count looks like for other users. Do you feel that it is weird to have a like count with the word "like" right next to the heart?

**Review by TheGuyWithoutH — COMMENTED** [review]
Small question of layout, otherwise it is straightforward and effective. Do not forget a little tests

File: app/src/main/java/ch/epfl/culturequest/social/PictureAdapter.java (hunk 2 above)
Comment (aelalamy42) [reply app/src/main/java/ch/epfl/culturequest/social/PictureAdapter.java:129]: I thought about it. I think either way is fine, but it still made more sense to me to have "likes" next to it.

**Review by TheGuyWithoutH — APPROVED** [review]
Seems good ! Just make sure the coverage is ok

**Pull request merged**